v2/handler.js

```diff
@@ -966,3 +971,33 @@ function InRequestOptions(
     self.hostPort = hostPort;
     self.tracer = tracer;
 }
+
+function validateHostPort(hostPort) {
+    var parts = hostPort.split(':');
+    if (parts.length !== 2) {
+        return false;
+    }
+
+    if (parts[0] === '127.0.0.1') {
+        return false;
+    }
+
+    if (parts[0] === '0.0.0.0') {
```

Comment:
@rf we need to allow 0.0.0.0 and 127.0.0.1

We can check again in hyperbahn advertise for 127.0.0.1